toolchain/llvm/llvm-compiler.in

```diff
@@ -16,8 +16,11 @@ if [ "$LTO" != "0" ] && [ "@CLANG_PACKAGES_LTO@" = "ON" ]; then
 fi
 
 if [ "$CONF" == "1" ]; then
-    SKIP_OPT="-O0 -fno-lto"
+    SKIP_OPT="-g0 -O0 -fno-lto -fno-data-sections -fno-function-sections"
 fi
 
-"$PROG" $LTO_FLAGS "$@" $FLAGS @CLANG_FLAGS@ @linker@ $SKIP_OPT
+if [ "$GC" != "0" ]; then
+    CLANG_GC="-fdata-sections -ffunction-sections -fno-unwind-tables -fomit-frame-pointer"
```

Comment:
Remove -fno-unwind-tables, -fomit-frame-pointer, -pipe flags

Reply:
That might true when compiling packages sequentially but not with this cmake system